src/Common/src/System/Net/Http/Http2/CopyToAspNetCore.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+if [ -n "$1" ]; then
+    remote_repo="$1"
+else
+    remote_repo="$ASPNETCORE_REPO"
+fi
+
+if [ -z "$remote_repo" ]; then
+    echo The 'ASPNETCORE_REPO' environment variable or command line paramter is not set, aborting.
```

Comment:
```suggestion
    echo The 'ASPNETCORE_REPO' environment variable or command line parameter is not set, aborting.
```

Reply:
Thanks. 

Copied that from https://github.com/dotnet/corefx/blob/70bdacc3e9fcb50acce9e4a3045e35452b5a3410/src/Common/src/System/Net/Http/Http2/CopyToAspNetCore.cmd#L7
Will fix this too.